src/io/flutter/view/OpenFlutterViewAction.java

```diff
@@ -39,7 +40,11 @@ public void actionPerformed(@NotNull final AnActionEvent e) {
 
     FlutterInitializer.sendAnalyticsAction(this);
 
-    final ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow("Flutter");
-    toolWindow.show(null);
+    final ToolWindow toolWindow = ToolWindowManager.getInstance(project).getToolWindow(FlutterView.TOOL_WINDOW_ID);
+    if (toolWindow == null) {
+      FlutterMessages.showError("Error Opening Flutter View", "Unable to find Flutter tool window");
```

Comment:
Perhaps modify the message in light of the above? `Unable to open View` and `Unable to open the Flutter tool window - no Flutter modules found.`